src/smallOrLargeNodes/content.md

```diff
@@ -0,0 +1,293 @@
+When you create a Kubernetes cluster, one of the first questions that pops up is: "what type of worker nodes should I use and how many of them?".
+
+If you're building an on-premises cluster, should you order some last-generation power servers, or use the dozen or so old machines that are lying around in your data centre?
+
+Or if you're using a managed Kubernetes service like [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/), should you use 8 `n1-standard-1` or 2 `n1-standard-4` instances to achieve your desired computing capacity?
+
+## Cluster capacity
+
+In general, a Kubernetes cluster abstracts a set of nodes as a single big node whose total compute capacity (in terms of CPU and memory) is the sum of all the constituent nodes' capacities.
+
+Naturally, there are thus multiple ways to achieve a desired target cluster capacity.
+
+For example, imagine that you need a cluster with a total capacity of 16 CPU cores and 16 GB of RAM.
+
+> For example, because the set of applications that you want to run on the cluster require this amount of resources.
+
+Here are just two of the possible ways to design your cluster:
+
+![Small vs. large nodes](small-vs-large-nodes.svg)
+
+Both options result in a cluster with the same capacity — but the left option uses 16 smaller nodes, whereas the right one uses 4 larger nodes.
+
+_Which is better?_
+
+To approach this question, let's look at the pros and cons of the two opposing directions of "few large nodes" and "many small nodes".
+
+> Note that "nodes" in this article specifically refers to worker nodes. The choice of number and size of master nodes is a different topic that should be treated separately.
+
+## Few large nodes
+
+The most extreme case in this direction would be to have a single worker node that provides the entire desired cluster capacity.
+
+In the above example, this would be a single worker node with 16 CPU cores and 16 GB of RAM.
+
+_Let's look at the advantages such an approach could have._
+
+### Pros
+
+#### Less management overhead
+
+Simply said, having to manage a small number of machines is less laborious than having to manage a large number of machines.
```

Comment:
I think I picked up machines from university. For example, "Kubernetes in Action" uses mostly machines too. Computers would be ok too. A server is a specific role of a computer, or it can also be just a piece of software. What's the point against machines?